database/migrations/2021_08_18_134409_create_schedules_table.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateSchedulesTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('schedules', function (Blueprint $table) {
+            $table->id();
+            $table->time('time');
```

Comment:
テーブル定義書かけてなかったんですが、ここは文字列型で大丈夫です！
```suggestion
            $table-> string('time');
```